mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 # Project information
-site_name: "Codacy Chart"
+site_name: codacy/chart
```

Comment:
Since this string appears on the documentation some customers are already using, I suggest we change it to something more user-friendly:

```suggestion
site_name: Chart
```

![image](https://user-images.githubusercontent.com/60105800/77685499-714d7a00-6f93-11ea-9267-19c60cdec971.png)